### PR TITLE
We use python 3.4,

### PR DESCRIPTION
**Pull request by captain-sysadmin**
Turns out there are two types on python3 on centos 7. So much joy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ### Local development
 
-* Requires Python 3, pip, psql client
+* Requires Python 3.4, pip, psql client
```
Comment (GeoffThorpeFT) [inline README.md]: I'm running 3.5.1 not 3.4

Comment (lucas42) [inline README.md]: Yes, ideally this should match the value in runtime.txt, because that's what'll be used in production.

Comment (captain-sysadmin) [inline README.md]: Aye, it doesn't work with 3.3 but it will work with 3.4,

**Comment by lucas42** [comment]
Looks fine to me :+1:

**Pull request merged**